.eslintignore

```diff
@@ -4,5 +4,6 @@
 /config/
 /dist/
 node_modules/
-scripts/utils/dist/
+/scripts/utils/dist/
+/scripts/utils/build/
```

Comment:
**question:** I don't understand why this is added. There is no configuration for this, and in any case, this is unrelated with the topic of this PR

Reply:
I don't remember. But I had a lot of times some files built in `build` directory or `JS` files directly in the source with my IDE. It's very annoying.

Reply:
OK let's keep it